lib/reducers/create-otp-reducer.js

```diff
@@ -306,6 +307,23 @@ function createOtpReducer(config) {
             ? filterProfileOptions(action.payload.response)
             : action.payload.response
         response.requestId = requestId
+        response.plan.itineraries = response.plan.itineraries.map(
+          (itinerary) => {
+            itinerary.legs = itinerary.legs.map((leg) => {
+              if (leg.routeId) {
+                leg.mode = checkForRouteModeOverride(
+                  {
+                    id: leg.routeId,
+                    mode: leg.mode
+                  },
+                  state?.config?.routeModeOverrides
```

Comment:
`state` is defined, so the `?` operator is not needed after `state`.

Reply:
Good catch thanks!